packages/client/src/pages/admin/pluggables/Reports.tsx

```diff
@@ -64,9 +66,28 @@ interface ReportsWidgetViewState {
 	showError: boolean;
 }
 
+// const isMaybeFlightStaff = pipe(
+// 	M.filterType<Member, CAPMember>(isCAPMember),
+// 	M.filter(hasOneDutyPosition(['Cadet Flight Commander', 'Cadet Flight Sergeant'])),
+// 	M.isSome,
+// );
+
+// const isFlightStaff = (member: Member | undefined | null): member is CAPMember =>
+// 	isMaybeFlightStaff(M.fromValue(member));
+
+// const hasAllowedDutyPosition = hasOneDutyPosition([
+// 	'Cadet Flight Commander',
+// 	'Cadet Flight Sergeant',
+// 	'Cadet Commander',
+// 	'Cadet Deputy Commander',
+// 	'Cadet Executive Officer',
+// 	'Deputy Commander For Cadets',
+// ]);
+
```

Comment:
Can be deleted, it's located in FlightContact.tsx

```suggestion

```

Reply:
It's a placeholder.  I'm not done with it yet.  I'll come back to it.